saleor/payment/gateways/stripe/stripe_api.py

```diff
@@ -198,8 +244,15 @@ def cancel_payment_intent(
         return payment_intent, None
     except StripeError as error:
         logger.warning(
-            "Unable to cancel a payment intent (%s), error", payment_intent_id
+            "Unable to cancel a payment intent",
+            extra={
+                "error_message": error.user_message,
+                "http_status": error.http_status,
+                "code": error.code,
+                "payment_intent": payment_intent_id,
+            },
```

Comment:
As `extra` data are repeated few times, maybe we should create some function for preparing this?

Reply:
Do you mean a function that will take params and create from it a dictionary? extra is repeated many times, but most of the calls have different arguments.